Octokit/Models/Response/EventInfo.cs

```diff
@@ -179,6 +179,26 @@ public enum EventInfoState
         Committed,
 
         /// <summary>
+        /// The actor requested review from the subject on this pull request.
+        /// </summary>
+        ReviewRequested,
+
+        /// <summary>
+        /// The actor dismissed a review from the pull request.
+        /// </summary>
+        ReviewDismissed,
+
+        /// <summary>
+        /// The actor removed the review request for the subject on this pull request.
+        /// </summary>
+        ReviewRequestRemoved,
+
+        /// <summary>
+        /// Not documented in GitHub API.
```

Comment:
I think we can advise upstream github support this value is missing from their docs, so rather than saying this I think we can just give it an actual description? 

Eg `Base branch of the pull request was changed`